flake.nix

```diff
@@ -77,6 +77,16 @@
         };
 
         formatter = pkgs.alejandra;
+
+        devShells.kernel = pkgs.mkShell {
+          packages = [pkgs.ncurses pkgs.pkg-config];
+          inputsFrom = [pkgs.linux_latest];
+          shellHook = ''
+            export src=${pkgs.linux_latest.src}
+            unpackPhase
```

Comment:
If we run unpackPhase I'm wondering should we also run patchPhase?

Reply:
Addressed in 898e9a7